# 3x3 homogenous 2d transformation matrix
float64[9] matrix
